### ISSUE TITLE
wgan code: typo in loss function

### ISSUE THREAD

**Issue by HashemMZ** [YeonwooSung/GAN_Implementation#5]
Hi, there is a typo in loss function of Disc:
`loss_D = - torch.mean(discriminator(real_imgs) + torch.mean(discriminator(fake_imgs)))`
the parenthesis of first mean() is closed at the end of the line which is wrong. due to this the network collapses fast. I tested the corrected code and it works with no problem:
`loss_D = - torch.mean(discriminator(real_imgs)) + torch.mean(discriminator(fake_imgs))`

**Comment by YeonwooSung** [comment]
Ah, Thanks a lot mate :)
Will apply it right now.
